Added bugfix release notes for MTM-58302 (#1919)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-core-10.20.313.0_Querying-inventory-fails-with-400-Failed-to-parse-JSON.md

---
date: ""

> title: Querying inventory fails with 400 Failed to parse JSON string

Reviewer: Please us a title which states what works well now

Author: Changed.

> title: Adding validation that prevents entering an empty key for custom attributes.

Reviewer: ```suggestion
title: Preventing creation of empty keys for custom attributes
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-58302
version: 10.20.313.0
---

> Added key validation for custom attributes to prevent creating empty keys in the inventory that cause issue with parsing when fetching data.

Reviewer: ```suggestion
In the inventory, key validation has been added for custom attributes to prevent creating empty keys which cause issues with parsing when fetching data.
```

Author: Changed.
